src/model/encoding/__tests__/encodeInlineStyleRanges-test.js

```diff
@@ -91,13 +91,13 @@ describe('encodeInlineStyleRanges', () => {
 
   it('must encode custom styles', () => {
     const custom = List([FOO, FOO, FOO_BAR, FOO_BAR, BOLD, BOLD]);
-    expect(
-      encodeInlineStyleRanges(createBlock(' '.repeat(6), custom)),
-    ).toEqual([
-      {offset: 0, length: 4, style: 'foo'},
-      {offset: 2, length: 2, style: 'bar'},
-      {offset: 4, length: 2, style: 'BOLD'},
-    ]);
+    expect(encodeInlineStyleRanges(createBlock(' '.repeat(6), custom))).toEqual(
```

Comment:
This was due to a new prettier been version been released, and was failing the ci. At least was only this that needed change!

After @vjeux  release a new version of prettier in www we should prob then pin draft.js to that version with a yarn.lock.